cpp/ps0000/problem solve 0.cpp

```diff
@@ -1,15 +1,16 @@
-#include <iostream>
+#include <iostream> // 2/2
 // اول غلطه كتب @ وهي مفروض #
 // تانى غلطة في كلمه "iostream.h  هي مفروض من غير علامتين تنصيص ومن غير .h
 using namespace std;
 //نسي يكتب الجمله دي
-int main()
+int main() // 1/1
 // كتب كلمه void 
 // كتب Main بدل main 
 {
-  cout << "Hello World \n " ;
+  cout << "Hello, World\n" ; // 3/4
 // حط دي >> مكان دي <<
 // كتب /n بدل \n

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [ ] output is as required `Hello, World!` in a single line